model/database/deprecated/metrics-deprecated.yaml

```diff
@@ -89,7 +102,8 @@ groups:
     type: metric
     metric_name: db.client.connections.wait_time
     stability: development
-    deprecated: "Replaced by `db.client.connection.wait_time`. Note: the unit also changed from `ms` to `s`."
+    deprecated:
+      reason: uncategorized
```

Comment:
do you want to add a note instead? Similar to what you did on the registry-deprecated?

Reply:
great catch, thanks!

Reply:
Can't we still cover it with the `renamed_to` and add an additional note? Isn't this way with only the note having the same limitations as before, with it being hard to do any code generation? Sorry if I missed something obvious

Reply:
I believe `renamed_to` is an also indication to automated schema conversion tools (not only for markdown generation)

Reply:
yes, the main goal is automated schema transformation, note is still there for humans